ds3-sdk/src/main/java/com/spectralogic/ds3client/serializer/XmlOutput.java

```diff
@@ -92,8 +92,8 @@ private static String toXml(final Object object, final FilterProvider filterProv
         }
     }
 
-    public static String toXml(final Ds3ObjectList objects, final BulkCommand command) {
-        if (command == BulkCommand.GET) {
```

Comment:
If we only care about BulkCommand.PUT, then I am confused why the code that I'm refactoring checks for BulkCommand.GET.  What am I missing?